conductor/submitter.py

```diff
@@ -238,8 +242,15 @@ def applyDefaultSettings(self):
         # Set the radio button on for the start/end frames by default
         self.on_ui_start_end_rdbtn_toggled(True)
 
-        # Set the default instance type
-        self.setInstanceType(self.default_instance_type)
+        # Set the default preemptible pref
+        default_preemptible = CONFIG.get('preemptible')
+        if default_preemptible != None:
```

Comment:
PEP8: `!= None` should be `is not None`.

Reply:
ha, that's what I was going to do, but figured you would say that "we're not comparing ids", see our discussion: https://github.com/AtomicConductor/conductor_ee/pull/172#discussion_r148578437

Reply:
addressed